pkg/webhooks/paddlepaddle/paddlepaddle_webhook.go

```diff
@@ -74,6 +75,7 @@ func validatePaddleJob(job *trainingoperator.PaddleJob) field.ErrorList {
 	if errors := apimachineryvalidation.NameIsDNS1035Label(job.Name, false); len(errors) != 0 {
 		allErrs = append(allErrs, field.Invalid(field.NewPath("metadata").Child("name"), job.Name, fmt.Sprintf("should match: %v", strings.Join(errors, ","))))
 	}
+	allErrs = util.ValidateManagedBy(&job.Spec.RunPolicy, allErrs)
```

Comment:
IIUC we miss validation that the value is immutable in `ValidateUpdate` functions.